source/scriptHandler.py

```diff
@@ -349,6 +358,7 @@ def script(
 	@param allowInSleepMode: Whether this script should run when NVDA is in sleep mode.
 	@param resumeSayAllMode: The say all mode that should be resumed when active before executing this script.
 	One of the C{sayAll.CURSOR_*} constants.
+	@param speakOnDemand: Either this script should speak when NVDA speech mode is "on demand"

```

Comment:
Could you make this sphinx style while at it?
```suggestion
	:param speakOnDemand: Either this script should speak when NVDA speech mode is "on demand"
```